lifelines/fitters/aalen_additive_fitter.py

```diff
@@ -183,11 +183,12 @@ def _fit_static(self, dataframe, duration_col, event_col=None,
             relevant_individuals = (ids == id)
             assert relevant_individuals.sum() == 1.
 
-            # perform linear regression step.
```

Comment:
the `for` loop above does nothing. I don't understand these changes

Reply:
Hmm, do you have a sketch of the algorithm for this? Now I'm confused, as it seemed that the linear regression step was causing this to hang when I ran it leading to the 4 days!!!.